app-misc/qlcplus/qlcplus-4.12.2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils udev virtualx xdg
```

Comment:
How is `udev` used?

Reply:
It's used to query the path where to put the udev rules here: https://github.com/gentoo/gentoo/pull/12418/files#diff-6045b184eacc87c5adb4beb2ec13ffe3R45